src/Umbraco.Core/Persistence/Repositories/IMediaRepository.cs

```diff
@@ -3,6 +3,10 @@
 
 namespace Umbraco.Core.Persistence.Repositories
 {
+    public interface IMediaRepository2 : IMediaRepository, IContentRepository2<int, IMedia>
```

Comment:
Make services implement v2 interface or move v2 interface methods to v1(prefer)

Reply:
This is one of those places in the codebase we are allowed to break interfaces, the db repositories and services - only additive changes allowed though.

Reply:
awesome

Reply:
updated to use existing interface